src/npm.ts

```diff
@@ -282,6 +283,43 @@ export const refreshPackageJsonData = (
   }
 }
 
+export const refreshPnpmWorkspaceData = (
```

Comment:
There is some duplication from `refreshPackageJsonData` which I could look into refactoring